templates/catalog/category_pagination.html

```diff
@@ -0,0 +1,19 @@
+{% comment %}
+@todo #302:30m Implement JS for pagination buttons.
+ Hide pagination buttons and reniew a canonical url for current page with the help of JS.
+ Currently buttons are hidden by CSS.
```

Comment:
It's dangerous to hide pagination with css.
Wrap pagination with `if debug` plz